cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -285,10 +285,9 @@ export class Application {
         selectItemsPerPage(100);
         cy.get(tdTag)
             .contains(this.name)
-            .parent(tdTag)
             .parent(trTag)
```

Comment:
Please use `.closest()` instead of `.parent()`, this will allow to be more flexible if page structure will be suddenly changed

Reply:
Done